.devcontainer/server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/go/.devcontainer/base.Dockerfile
+
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.19, 1.18, 1-bullseye, 1.19-bullseye, 1.18-bullseye, 1-buster, 1.19-buster, 1.18-buster
+ARG VARIANT="1.19-bullseye"
+FROM mcr.microsoft.com/vscode/devcontainers/go:0-${VARIANT}
+
+# [Choice] Node.js version: none, lts/*, 18, 16, 14
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+RUN go install golang.org/dl/go1.19.1@latest \
+&& go1.19.1 download 
```

Comment:
I don't think these two lines (10-11) are necessary as the base image itself comes with both go and its toolchain installed

Reply:
Good feedback for @sivanaikk’s consideration.

Reply:
@sivanaikk what do you think?

Reply:
Previously we were using go1.19.1 in Makefile. Have to update Dockerfile